compute/gpu/reference-content/choosing-gpu-instance-type.mdx

```diff
@@ -33,27 +33,70 @@ Below, you will find a guide to help you make an informed decision:
+|------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+|   GPU Type                                     |   [L40S 48GB PCIe](https://www.nvidia.com/en-us/data-center/l40s/)                                                                                                               |   NVIDIA [GH200 Grace Hopper™ Superchip](https://www.nvidia.com/en-us/data-center/grace-hopper-superchip/)                                                                                                                                                  |
+|   NVIDIA architecture                          |   Lovelace 2022                                                                                                                                                                  |   GH200 Grace Hopper™ Architecture                                                                                                                                                                                                                          |
+|   Performance (training in FP16 Tensor Cores)  |   Up to 362 TFLOPS                                                                                                                                                               |   Up to 989 peak TFLOPs per GH200                                                                                                                                                                                                                           |
+|   Specifications                               |   under construction                                                                                                                                                             |   - GH200 SuperChip with 72 ARM Neoverse V2 cores alongside 480 GB of LPDDR5X DRAM and 96GB of HBM3 GPU memory (Memory is fully merged for up to 576GB of global usable memory).<br />- 1,92TB of scratch storage<br />- Up to 25GBps of networking performances |
```

Comment:
Are the "under construction" parts meant to be there?

Reply:
Yes, the offer details are not clear yet